support-frontend/assets/components/subscriptionCheckouts/subscriptionSubmitButton.jsx

```diff
@@ -20,28 +19,34 @@ import './subscriptionSubmitButton.scss';
 type PropTypes = {|
   paymentMethod: Option<PaymentMethod>,
   allErrors: FormError<FormField>[],
+  className?: Option<string>,
+  component: Node,
 |};
 
 // ----- Render ----- //
 
 
 function SubscriptionSubmitButton(props: PropTypes) {
   return (
-    <span>
-      {props.paymentMethod !== PayPal && (
+    <span className={props.paymentMethod === props.className ? 'show' : 'hide'}>
       <div className="component-submit-button">
-        <div className="component-submit-button--margin">
+        <div className={props.paymentMethod === DirectDebit ? 'component-submit-button--margin' : ''}>
```

Comment:
Would be good to find a nicer way to do this, and the same for the PayPal submit button above.